web/.github-issues/setup-devshell-and-tools.md

```
@@ -0,0 +1,152 @@
+  ```bash
+  cd web
+  pnpm init
+  pnpm add react react-dom react-router-dom
+  pnpm add -D @types/react @types/react-dom typescript vite @vitejs/plugin-react
+  pnpm add -D @biomejs/biome prettier
+  ```
+
+#### プロジェクト構造の作成
+
+- [x] プロジェクト構造作成完了
+  ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)